projects/igniteui-angular/src/lib/grids/grid/column.spec.ts

```diff
@@ -359,3 +391,23 @@ export class ColumnHaederClassesComponent {
     @ViewChild(IgxGridComponent, { read: IgxGridComponent })
     public grid: IgxGridComponent;
 }
+
```

Comment:
Could we re-use some of the existing components?

Reply:
Existing component now used